kombu/serialization.py

```diff
@@ -410,24 +410,26 @@ def not_available(*args, **kwargs):
     'application/x-msgpack': register_msgpack,
 }
 
+_noset = object()
```

Comment:
Would you mind reconsidering the name? Perhaps it should uppercase as a constant?

Reply:
Sure happy to change to whatever. `NOTSET` then?

Reply:
Sounds good!